swagger-restapi-client-base/pom.xml

```diff
@@ -88,6 +84,21 @@
             <groupId>com.brsanthu</groupId>
             <artifactId>migbase64</artifactId>
         </dependency>
+
+        <!-- Annotations used by the generated swagger code -->
+        <dependency>
+            <groupId>jakarta.annotation</groupId>
+            <artifactId>jakarta.annotation-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <!-- Required to avoid a 'Used undeclared dependencies found' error from the maven-dependency-plugin -->
```

Comment:
This same dependency is found when I run `mvn dependency:tree` on the java 8 version of tag-service-analytics-client (left hand side) and it is also found in the dependency tree for the Java 11 version of tag-service-analytics-client (right hand side). However, it needs to be explicitly declared on the Java 11 version, otherwise I get an 'Used undeclared dependency error':

![image](https://user-images.githubusercontent.com/55391479/149973719-96731d53-9b7e-4fd3-8bda-743b0b6524b0.png)